### ISSUE TITLE
Using *_VOLUMES on windows

### ISSUE THREAD

**Issue by andyneff** [VisionSystemsInc/vsi_common#30]
The _VOLUMES feature in docker on windows does not working after 17.12

It is unclear as to the exact cause of the issue.

In powershell:

```bash
docker run -it --rm -v /d/src/test:/test1 debian:9 # Works
docker run -it --rm -v D:\src\test:/test1 debian:9 # Works

docker run -it --rm -v D\src\test:/test1 debian:9 # Does not work, nor is it expected to
```

- Docker 17.12.0, docker-compose 1.18.0 (works)

```yaml
volumes:
- D:\src\test:/test1 # Works, but just will erroneously create a D directory
- /d/src/test:/test2  # Works
```

- Docker 18.03.0, docker-compose 1.20.1

```yaml
volumes:
- D:\src\test:/test1 # Works, but just will erroneously create a D directory
- /d/src/test:/test2  # Does NOT work
```

Mixing Docker 17.12.0 with docker-compose 1.18.0, still result in `/d/src/test:/test2` not working, only with a different error message

```
The source path "\\d\\src\\test:/test2"
is not a valid Windows path
```

@drewgilliam

**Comment by drewgilliam** [comment]
Issue reported on docker/compose
https://github.com/docker/for-win/issues/1829

workaround (setting COMPOSE_CONVERT_WINDOWS_PATHS=1) appears to work

**Comment by andyneff** [comment]
Between versions (1.20.0, 1.22.0) (non-inclusive), windows will have "Cannot locate specified dockerfile" errors. Should be fixed in 1.22.0 (docker compose issue 5969)

**Comment by andyneff** [comment]
There still appears to be some issues here

1. New just should use `VSI_PATH_ESC`
2. `docker_compose_override` should use `cygpath -w` if windows.

I think that will resolve this once and for all, without needing `COMPOSE_CONVERT_WINDOWS_PATHS`
